src/main/java/org/violetmoon/zeta/config/ConfigManager.java

```diff
@@ -171,21 +174,25 @@ public boolean isCategoryEnabled(ZetaCategory cat) {
         return enabledCategories.contains(cat);
     }
 
-    // ummm
-
     public ConfigFlagManager getConfigFlagManager() {
         return cfm;
     }
 
+    //whats this for?
+    public void addOnReloadListener(String id, Consumer<IZetaConfigInternals> consumer) {
```

Comment:
I think it was for the "hide removed items from JEI" option? Safe to delete, no usages in Quark atm